kyuubi-server/src/main/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilder.scala

```diff
@@ -212,9 +220,12 @@ class FlinkProcessBuilder(
 }
 
 object FlinkProcessBuilder {
-  final val FLINK_EXEC_FILE = "flink"
+  final val FLINK = "flink"
   final val APP_KEY = "flink.app.name"
```

Comment:
why this has a `flink.` suffix, but others does not?

Reply:
> why this has a `flink.` suffix, but others does not?

Because this is not a valid flink conf, it should be:

https://github.com/apache/kyuubi/blob/17df0844d06c5104f45735cf35dc45c0549253b3/kyuubi-server/src/main/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilder.scala#L227